packages/core/lib/core.dart

```diff
@@ -0,0 +1,6 @@
+/// Core components for Widgetbook Demo App
+library core;
+
+export 'src/styles/styles.dart';
+export 'src/tweet/tweet.dart';
+export 'src/widgets/widgets.dart';
```

Comment:
I like to have leading `.` in front of the src path since VS Code is then auto completing paths on adjusting it. 

`export 'src/styles/styles.dart';` would become `export './src/styles/styles.dart';`

Reply:
What do you think?

Reply:
Not sure I understand correctly but I can autocomplete in InteliJ without adding `.`. But I'll add it anyway if it will allow autocomplete in VS Code